test/aircraft/Pilot/maintainAltitude.spec.js

```diff
@@ -5,133 +5,161 @@ import _isArray from 'lodash/isArray';
+import { ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK } from '../_mocks/aircraftMocks';
+
 
 ava('.maintainAltitude() should set mcp.altitude to the correct value when greater than maxAssignableAltitude', (t) => {
-    const currentAltitudeMock = 5000;
     const nextAltitudeMock = 100000;
     const shouldExpediteMock = false;
     const shouldUseSoftCeilingMock = true;
     const pilot = new Pilot(modeControllerFixture, fmsArrivalFixture);
+    const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, navigationLibraryFixture);
 
     pilot.maintainAltitude(
-        currentAltitudeMock,
         nextAltitudeMock,
         shouldExpediteMock,
         shouldUseSoftCeilingMock,
-        airportModelFixture
+        airportModelFixture,
+        model
     );
 
     t.true(pilot._mcp.altitudeMode === 'HOLD');
-    t.true(pilot._mcp.altitude === 19001);
+    t.true(pilot._mcp.altitude === model.altitude);
 });
 
 ava('.maintainAltitude() should set mcp.altitudeMode to `HOLD` and set mcp.altitude to the correct value', (t) => {
-    const currentAltitudeMock = 5000;
     const nextAltitudeMock = 13000;
     const shouldExpediteMock = false;
     const shouldUseSoftCeilingMock = false;
     const pilot = new Pilot(modeControllerFixture, fmsArrivalFixture);
+    const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, navigationLibraryFixture);
 
     pilot.maintainAltitude(
-        currentAltitudeMock,
         nextAltitudeMock,
         shouldExpediteMock,
         shouldUseSoftCeilingMock,
-        airportModelFixture
+        airportModelFixture,
+        model
     );
 
     t.true(pilot._mcp.altitudeMode === 'HOLD');
     t.true(pilot._mcp.altitude === 13000);
 });
 
 ava('.maintainAltitude() calls .shouldExpediteAltitudeChange() when shouldExpedite is true', (t) => {
-    const currentAltitudeMock = 5000;
     const nextAltitudeMock = 13000;
     const shouldExpediteMock = true;
     const shouldUseSoftCeilingMock = false;
     const pilot = new Pilot(modeControllerFixture, fmsArrivalFixture);
     const shouldExpediteAltitudeChangeSpy = sinon.spy(pilot, 'shouldExpediteAltitudeChange');
+    const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, navigationLibraryFixture);
 
     pilot.maintainAltitude(
-        currentAltitudeMock,
         nextAltitudeMock,
         shouldExpediteMock,
         shouldUseSoftCeilingMock,
-        airportModelFixture
+        airportModelFixture,
+        model
     );
 
     t.true(shouldExpediteAltitudeChangeSpy.calledOnce);
 });
 
 ava('.maintainAltitude() returns the correct response strings when shouldExpedite is false', (t) => {
-    const currentAltitudeMock = 5000;
     const nextAltitudeMock = 13000;
     const shouldExpediteMock = false;
     const shouldUseSoftCeilingMock = false;
     const pilot = new Pilot(modeControllerFixture, fmsArrivalFixture);
+    const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK, navigationLibraryFixture);
```

Comment:
`aircraftModel` contains a Pilot and MCP. Since we are now creating the entire aircraft, we must remove the `pilot` const above, and use `aircraftModel.pilot` in all below. This is the case in multiple tests.